libs/fluid-elements/checkbox/src/lib/checkbox.ts

```diff
@@ -86,6 +86,10 @@ export class FluidCheckbox extends LitElement {
   })
   disabled = false;
 
+  /** The value attribute of the native input element */
+  @property({ type: String, reflect: false })
```

Comment:
The value can be reflected, it is also when done natively.
```suggestion
  @property({ type: String, reflect: true })
```